pandas/io/excel.py

```diff
@@ -97,8 +97,10 @@ def read_excel(io, sheetname=0, **kwds):
         * [0,1,"Sheet5"] -> 1st, 2nd & 5th sheet as a dictionary of DataFrames
         * None -> All sheets as a dictionary of DataFrames
 
-    header : int, default 0
+    header : int, list of ints, default 0
         Row to use for the column labels of the parsed DataFrame
+        If a list of integers is passed those row positions will
```

Comment:
maybe say they start numbering from 0, right? (it should be obvious, but this is excel....)